### PR TITLE
create new column to indicate if dataset is tiled

### LINKED ISSUE #99: The asterisk in the resolution codes is misleading
### Description of the problem

Hello guys! When I use `grid = pygmt.datasets.load_earth_relief(resolution="06m", region=[70, 140, 0, 55])` to set a map of map, I found the provided resolution code can not be used.

### Minimal Complete Verifiable Example

```python
For example, when I set the resolution as "03m*", as codes as :

grid = pygmt.datasets.load_earth_relief(resolution="03m*", region=[70, 140, 0, 55])
fig = pygmt.Figure()
fig.grdimage(grid=grid, projection="M15c", frame="a", cmap="geo")
fig.colorbar(frame=["a1000", "x+lElevation", "y+lm"])
fig.show()
```

### Full error message

```python-traceback
---------------------------------------------------------------------------
GMTInvalidInput                           Traceback (most recent call last)
Cell In[21], line 1
----> 1 grid = pygmt.datasets.load_earth_relief(resolution="03m*", region=[70, 140, 0, 55])
      2 fig = pygmt.Figure()
      3 fig.grdimage(grid=grid, projection="M15c", frame="a", cmap="geo")

File f:\Anaconda3\envs\GeoTable\lib\site-packages\pygmt\helpers\decorators.py:738, in kwargs_to_strings..converter..new_module(*args, **kwargs)
    736             kwargs[arg] = separators[fmt].join(f"{item}" for item in value)
    737 # Execute the original function and return its output
--> 738 return module_func(*args, **kwargs)

File f:\Anaconda3\envs\GeoTable\lib\site-packages\pygmt\datasets\earth_relief.py:159, in load_earth_relief(resolution, region, registration, data_source, use_srtm)
    156 else:
    157     dataset_prefix = earth_relief_sources[data_source]
--> 159 grid = _load_remote_dataset(
    160     dataset_name="earth_relief",
    161     dataset_prefix=dataset_prefix,
    162     resolution=resolution,
    163     region=region,
    164     registration=registration,
    165 )
    166 return grid

File f:\Anaconda3\envs\GeoTable\lib\site-packages\pygmt\helpers\decorators.py:738, in kwargs_to_strings..converter..new_module(*args, **kwargs)
...
    279 # check registration
    280 if registration is None:
    281     # use gridline registration unless only pixel registration is available

GMTInvalidInput: Invalid resolution '03m*'.
```

### System information

```bash
PyGMT information:
  version: v0.10.0
System information:
  python: 3.10.13 | packaged by Anaconda, Inc. | (main, Sep 11 2023, 13:24:38) [MSC v.1916 64 bit (AMD64)]
  executable: f:\Anaconda3\envs\GeoTable\python.exe
  machine: Windows-10-10.0.19041-SP0
Dependency information:
  numpy: 1.26.1
  pandas: 2.1.1
  xarray: 2023.10.1
  netCDF4: 1.6.5
  packaging: 23.2
  contextily: None
  geopandas: None
  IPython: 8.17.2
  rioxarray: None
  ghostscript: 10.02.0
GMT library information:
  binary version: 6.4.0
  cores: 12
  grid layout: rows
  image layout: 
  library path: F:/Anaconda3/envs/GeoTable/Library/bin/gmt.dll
  padding: 2
  plugin dir: F:/Anaconda3/envs/GeoTable/Library/bin/gmt_plugins
  share dir: F:/Anaconda3/envs/GeoTable/Library/share/gmt
  version: 6.4.0
```

### PR DESCRIPTION
**Pull request by anbj**
Remove asterisk and create table to indicate tiled datasets.

Should close #99.

### CODE REVIEW

**Comment by anbj** [comment]
fyy: got some errors which I think needs attention:

```
/home/anbj/remote-datasets/docs/earth-faaerror.rst:: WARNING: duplicate label tbl-earth_faa, other instance in /home/anbj/remote-datasets/docs/earth-faa.rst
/home/anbj/remote-datasets/docs/earth-faaerror.rst:: WARNING: image file not readable: _static/GMT_earth_faaerror.jpg
checking consistency... /home/anbj/remote-datasets/docs/earth-faaerror.rst: WARNING: document isn't included in any toctree
```

File: docs/earth-age.rst [hunk 1]
```diff
@@ -25,23 +25,23 @@ for pixel-registered grids; gridline-registered grids increment dimensions by on
 
 .. _tbl-earth_age:
 
-.. table:: EarthByte Global Earth Seafloor Crustal Age. An asterisk denotes tiled datasets.
+.. table:: EarthByte Global Earth Seafloor Crustal Age.
```
Comment (yvonnefroehlich) [inline docs/earth-age.rst:28]: Not directly related to this PR, but as "Global" was removed from the names of the remote datasets in PR #98, maybe it should be removed also here? 
```suggestion
.. table:: EarthByte Earth Seafloor Crustal Age.
```

File: docs/earth-faa.rst [hunk 2]
```diff
@@ -28,23 +28,23 @@ for pixel-registered grids; gridline-registered grids increment dimensions by on
 
 .. _tbl-earth_faa:
 
-.. table:: IGPP Global Earth Free-Air Anomaly. An asterisk denotes tiled datasets.
+.. table:: IGPP Global Earth Free-Air Anomaly.
```
Comment (yvonnefroehlich) [inline docs/earth-faa.rst:31]:
```suggestion
.. table:: IGPP Earth Free-Air Anomaly.
```

File: docs/earth-faaerror.rst [hunk 3]
```diff
@@ -27,23 +27,23 @@ for pixel-registered grids; gridline-registered grids increment dimensions by on
 
 .. _tbl-earth_faa:
 
-.. table:: IGPP Global Earth Free-Air Anomaly. An asterisk denotes tiled datasets.
+.. table:: IGPP Global Earth Free-Air Anomaly.
```
Comment (yvonnefroehlich) [inline docs/earth-faaerror.rst:30]:
```suggestion
.. table:: IGPP Earth Free-Air Anomaly.
```

File: docs/earth-gebco.rst [hunk 4]
```diff
@@ -28,27 +28,27 @@ for pixel-registered grids; gridline-registered grids increment dimensions by on
...
-  04m*     5400 x   2700 g,p   17 MB  4 arc minute global relief (GEBCO 2023 @ 7.5 km)
-  03m*     7200 x   3600 g,p   30 MB  3 arc minute global relief (GEBCO 2023 @ 5.6 km)
-  02m*    10800 x   5400 g,p   65 MB  2 arc minute global relief (GEBCO 2023 @ 3.7 km)
-  01m*    21600 x  10800 g,p  237 MB  1 arc minute global relief (GEBCO 2023 @ 1.9 km)
-  30s*    43200 x  21600 g,p  864 MB  30 arc second global relief (GEBCO 2023 @ 1.0 km)
-  15s*    86400 x  43200 p    2.9 GB  15 arc second global relief (GEBCO 2023)
-  03s*   432000 x 216000 g    6.8 GB  3 arc second global relief (SRTM3S)
-  01s*  1296000 x 432000 g     41 GB  1 arc second global relief (SRTM1S)
-  ==== ================= === =======  ================================================
+.. table:: GEBCO Global Earth Relief.
```
Comment (yvonnefroehlich) [inline docs/earth-gebco.rst:31]:
```suggestion
.. table:: GEBCO Earth Relief.
```

File: docs/earth-geoid.rst [hunk 5]
```diff
@@ -20,23 +20,23 @@ for pixel-registered grids; gridline-registered grids increment dimensions by on
...
-  15m      1440 x    720 g,p  1.1 MB  15 arc minute global EGM2008 geoid (1 min @ 28 km)
-  10m      2160 x   1080 g,p  2.9 MB  10 arc minute global EGM2008 geoid (1 min @ 18 km)
-  06m      3600 x   1800 g,p  5.9 MB  6 arc minute global EGM2008 geoid (1 min @ 10 km)
-  05m*     4320 x   2160 g,p  8.1 MB  5 arc minute global EGM2008 geoid (1 min @ 9 km)
-  04m*     5400 x   2700 g,p   11 MB  4 arc minute global EGM2008 geoid (1 min @ 7.5 km)
-  03m*     7200 x   3600 g,p   20 MB  3 arc minute global EGM2008 geoid (1 min @ 5.6 km)
-  02m*    10800 x   5400 g,p   34 MB  2 arc minute global EGM2008 geoid (1 min @ 3.7 km)
-  01m*    21600 x  10800   g  100 MB  1 arc minute global EGM2008 geoid (1 min original)
-  ==== ================= === =======  ==================================================
+.. table:: EGM2008 Global Earth Geoid.
```
Comment (yvonnefroehlich) [inline docs/earth-geoid.rst:23]:
```suggestion
.. table:: EGM2008 Earth Geoid.
```

File: docs/earth-mag.rst [hunk 6]
```diff
@@ -28,22 +28,22 @@ refers to the ``earth_mag4km`` version (the oceanic files are ~60% smaller):
 
 .. _tbl-earth_mag:
 
-.. table:: EMAG2 Global Earth Magnetic Anomaly Model. An asterisk denotes tiled datasets.
+.. table:: EMAG2 Global Earth Magnetic Anomaly Model.
```
Comment (yvonnefroehlich) [inline docs/earth-mag.rst:31]:
```suggestion
.. table:: EMAG2 Earth Magnetic Anomaly Model.
```

File: docs/earth-relief.rst [hunk 7]
```diff
@@ -30,27 +30,27 @@ for pixel-registered grids; gridline-registered grids increment dimensions by on
...
-  04m*     5400 x   2700 g,p   17 MB  4 arc minute global relief (SRTM15+V2.5.5 @ 7.5 km)
-  03m*     7200 x   3600 g,p   30 MB  3 arc minute global relief (SRTM15+V2.5.5 @ 5.6 km)
-  02m*    10800 x   5400 g,p   65 MB  2 arc minute global relief (SRTM15+V2.5.5 @ 3.7 km)
-  01m*    21600 x  10800 g,p  237 MB  1 arc minute global relief (SRTM15+V2.5.5 @ 1.9 km)
-  30s*    43200 x  21600 g,p  864 MB  30 arc second global relief (SRTM15+V2.5.5 @ 1.0 km)
-  15s*    86400 x  43200 p    2.9 GB  15 arc second global relief (SRTM15+V2.5.5)
-  03s*   432000 x 216000 g    6.8 GB  3 arc second global relief (SRTM3S)
-  01s*  1296000 x 432000 g     41 GB  1 arc second global relief (SRTM1S)
-  ==== ================= === =======  ==================================================
+.. table:: IGPP Global Earth Relief.
```
Comment (yvonnefroehlich) [inline docs/earth-relief.rst:33]:
```suggestion
.. table:: IGPP Earth Relief.
```

File: docs/earth-vgg.rst [hunk 8]
```diff
@@ -27,23 +27,23 @@ for pixel-registered grids; gridline-registered grids increment dimensions by on
 
 .. _tbl-earth_vgg:
 
-.. table:: IGPP Global Earth Vertical Gravity Gradient. An asterisk denotes tiled datasets.
+.. table:: IGPP Global Earth Vertical Gravity Gradient.
```
Comment (yvonnefroehlich) [inline docs/earth-vgg.rst:30]:
```suggestion
.. table:: IGPP Earth Vertical Gravity Gradient.
```

**Comment by PaulWessel** [comment]
Will be lots of conflicts after all the new data set as well as updates sizes are merged (#102) so I think the best solution is to not approve this PR and if the desire is still there to flag the tiles then redo another PR later.

**Comment by seisman** [comment]
After thinking twice, I think it's fine to remove the tiled/untiled information from the table.

**Comment by PaulWessel** [comment]
I think we decided not to add the column since it is not needed for the user and can lead to confusion.

**Comment by anbj** [comment]
Good. Closing.

**Pull request closed without merging**